spec.html

```diff
@@ -28312,238 +28312,246 @@
+                  <td>
+                    `J`
+                  </td>
+                  <td>
+                    `K`
+                  </td>
+                  <td>
+                    `L`
+                  </td>
+                  <td>
+                    `M`
+                  </td>
+                  <td>
+                    `N`
+                  </td>
+                  <td>
+                    `O`
+                  </td>
+                  <td>
+                    `P`
+                  </td>
+                  <td>
+                    `Q`
+                  </td>
+                  <td>
+                    `R`
+                  </td>
+                  <td>
+                    `S`
+                  </td>
+                  <td>
+                    `T`
+                  </td>
+                  <td>
+                    `U`
+                  </td>
+                  <td>
+                    `V`
+                  </td>
+                  <td>
+                    `W`
+                  </td>
+                  <td>
+                    `X`
+                  </td>
+                  <td>
+                    `Y`
+                  </td>
+                  <td>
+                    `Z`
+                  </td>
+                </tr>
+                <tr>
+                  <td>
+                    `0`
+                  </td>
+                  <td>
+                    `1`
+                  </td>
+                  <td>
+                    `2`
+                  </td>
+                  <td>
+                    `3`
+                  </td>
+                  <td>
+                    `4`
+                  </td>
+                  <td>
+                    `5`
+                  </td>
+                  <td>
+                    `6`
+                  </td>
+                  <td>
+                    `7`
+                  </td>
+                  <td>
+                    `8`
+                  </td>
+                  <td>
+                    `9`
+                  </td>
+                  <td>
+                    `_`
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                  <td>
+                  </td>
+                </tr>
+              </tbody>
+            </table>
+          </figure>
+          1. If  _Unicode_ is *true*, then
```

Comment:
I thought about eliminating the _Unicode_ check, however it shows that the change is there for Unicode Word character matching.